packages/hardhat-core/src/internal/cli/project-creation.ts

```diff
@@ -271,11 +185,9 @@ async function printRecommendedDepsInstallationInstructions(
 async function writeEmptyHardhatConfig() {
   return fsExtra.writeFile(
     "hardhat.config.js",
-    `/**
- * @type import('hardhat/config').HardhatUserConfig
- */
+    `/** @type import('hardhat/config').HardhatUserConfig */
 module.exports = {
-  solidity: "${DEFAULT_SOLC_VERSION}",
+  solidity: "0.8.9",
```

Comment:
This on purpose? Seems like something we'd want to keep parameterized.

Reply:
I de-parametrized it to be consistent with the rest of the sample projects. We need to modify our "update solidity versions" doc to include this.

Reply:
Note that we can be more reckless here and just bump it whenever we want.